src/components/sequencing/SequenceEditor.svelte

```diff
@@ -203,11 +219,17 @@
                     parsedChannelDictionary,
                     parsedCommandDictionary,
                     nonNullParsedParameterDictionaries,
+                    librarySequences,
                   ),
                 ),
               ),
               compartmentSeqLinter.reconfigure(
-                inputLinter(parsedChannelDictionary, parsedCommandDictionary, nonNullParsedParameterDictionaries),
+                inputLinter(
```

Comment:
We'll probably want to pass these sequences to the adaptation as well but that doesn't need to be included in this PR.

Reply:
I agree, I wasn't sure how the adaptation will use this yet. We can investigat this more when we have library imports